Разметка всех страниц

daniilsakharov69 opened the pull request:
Я сделаль, надеюсь, нормально
¯ \ _ (ツ) _ / ¯

---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/38/user/925099/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/925099-sedona-38/2/

BlobJoe committed:
Задание: Разметка всех страниц

Я сделаль, надеюсь, нормально
¯ \ _ (ツ) _ / ¯

BlobJoe committed:
Правки в ссылках

Небольшое исправление ошибки в ссылках (случайно указал в <img>, а не в <a>

BlobJoe committed:
Ещё одна маленькая правка

Маленькое исправление (отсутствие заглушки)

eepodoprigora commented:
Привет, есть пару правок:
1. В хэдере иконка поиска это кнопка ;
2.  h1  Sedona . Лучше писать чуть подробнее о смысле заголовка, например, Седона - лучший курорт в Аризоне;
3. В секции преимуществ картинки нужно сделать отдельными лишками в списке с карточками в тексте;
4.  В секции удобств картинка идет первой в карточке и в разметке тоже должна быть первой;

сттраница Каталог:
1. Для формы есть тег fieldset, в данном контексте он лучше подойдет;
2.  Для звездочек в карточках могут потребоваться дополнительные теги в будущем;

В целом, отлично!

BlobJoe committed:
Правки из комментария

Привет!
Внёс правки, но не очень понял 3-й пункт в index'е.
Подскажи, пожалуйста, ты имеешь в виду отдельно обернуть каждую картинку в преимуществах в теги <ul> -> <li>?

BlobJoe committed:
Мини-правки

Исправил небольшие недочёты

BlobJoe committed:
Update catalog.html

eepodoprigora commented:
Привет, в преимуществах структура должна выглядеть так
![image](https://github.com/htmlacademy-htmlcss/925099-sedona-38/assets/89517901/5c83c7c4-9458-48e7-9dd9-84ca3f4572a0)
